doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3195,10 +3197,10 @@ hints of the database named core.
 The following options enable printing of informative or debug information for
 the :tacn:`auto` and :tacn:`trivial` tactics:
 
-.. opt:: Info Auto
-.. opt:: Debug Auto
-.. opt:: Info Trivial
-.. opt:: Debug Trivial
+.. flag:: Info Auto
+.. flag:: Debug Auto
+.. flag:: Info Trivial
+.. flag:: Debug Trivial
```

Comment:
When there are such a series of same objects you can group them with:

```
.. flag:: Info Auto
          Debug Auto
          Info Trivial
          Debug Trivial
   :undocumented:
```
The last `:undocumented:` bit signaling that the body is missing "on purpose".